### ISSUE TITLE
Bug: Tooltip size is altered in strange ways when space availability usually rather causes a re-orientation of position

### ISSUE THREAD

**Issue by k290** [ng-matero/extensions#232]
When out of space, the extension tooltip:

1.) Often changes size to a very awkward dimension. I can reproduce this in all possible position settings (but not consistently). 
2.) Size remains with strange dimension even when size is returned.

The correct functionality is to always change the orientation of the tooltip, which is how it seems to work 50% of the time.

See below video of the issue:
https://youtu.be/4i1dxGhQ0_Y

**Comment by nzbin** [comment]
I know this issue. It‘s because there has two different tooltips (`mat-tooltip` and `mtx-tooltip`). The component CSS has conflicts. In the real app, it should use only one type of tooltip and shouldn't mixes use. It also can be fixes with following CSS. But I'm not sure what the side effect.

```css
.mat-mdc-tooltip {
  position: relative !important;
}
```

**Comment by k290** [comment]
Ah I see..

The `.mat-mdc-tooltip` sometimes has higher priority

If your styles have higher [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) then they will take preference.

A solution that doesn't interfere with `.mat-mdc-tooltip` is

```
div. mtx-mdc-tooltip{
    position: relative;
    transform: scale(0);
}
````

Similarly, if I make the same change to your source code [here](https://github.com/ng-matero/extensions/blob/33bda632bc66ce818f04c335137b0dcb1b1b272e/projects/extensions/tooltip/tooltip.scss#L20) then it also takes precedence:

![image](https://github.com/ng-matero/extensions/assets/5611876/cc6094c5-2474-4c32-8645-1a104c3dad2b)

**Comment by nzbin** [comment]
If you hover the `mat-tooltip` first and then hover the `mtx-tooltip`. The `mat-tooltip` will have the same problems. I don't want modify the CSS because I don't know the side effect.

**Comment by nzbin** [comment]
I have fixed this issue, please try the new version 16.0.3.

**Comment by k290** [comment]
Thank you @nzbin . Much better
